llvm-external-projects/iree-dialects/lib/Dialect/LinalgExt/Utils/Utils.cpp

```diff
@@ -17,6 +18,10 @@ namespace IREE {
 namespace LinalgExt {
 
 Value getDimValue(OpBuilder &builder, Location loc, Value v, int64_t dim) {
+  ShapedType type = v.getType().cast<ShapedType>();
+  if (!type.isDynamicDim(dim)) {
+    return builder.create<arith::ConstantIndexOp>(loc, type.getDimSize(dim));
+  }
```

Comment:
The tiling lit tests have to be fixed because of this change. This is needed to let TileUsingInterface use attributes for sizes.